tree-sitter/README.md

```diff
@@ -1,6 +1,10 @@
 # haskell-tree-sitter
 
-**Please note that this library is currently in flux. The maintainers are revising the core API to allow for generation of Haskell types from a tree-sitter grammar, and the automatic bridging of tree-sitter syntax nodes to these types.**
```

Comment:
I think we can say it's stable enough to not warrant this message anymore? Happy leaving it in there too, given the precise AST explorations in Semantic may bring some new insights about what needs to change. Nothing should be major, though.